# 节点名称，必须与 launch 文件中的 name 一致
/target_ekf_node:
  ros__parameters:
    # 相机内参
    cam_width: 640.0
    cam_height: 480.0
    cam_fx: 346.74048
    cam_fy: 349.12708
    cam_cx: 322.83899
    cam_cy: 234.54992
    
    # 相机外参 (旋转矩阵 3x3 展平)
    # 0.0,  0.0,  1.0,
    # -1.0,  0.0,  0.0,
    # 0.0, -1.0,  0.0
    cam2body_R: [0.0, 0.0, 1.0, -1.0, 0.0, 0.0, 0.0, -1.0, 0.0]
    
    # 相机外参 (平移向量)
    cam2body_p: [0.0, 0.0, 0.1]
    
    # 其他默认参数
    check_fov: false
    ekf_rate: 20
